modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java

```diff
@@ -308,6 +460,31 @@ private Boolean getTranslationMode(MediaPackage mediaPackage, WorkflowInstance w
     return translateMode;
   }
 
+  /**
+   * Gets the 'limit to one' configuration
+   *
+   * @param mediaPackage     Contains mediapackage information
+   * @param workflowInstance Contains the workflow configuration
+   * @return Boolean if only max one track shall be transcribed
+   */
+  private Boolean getLimitToOneConfig(MediaPackage mediaPackage, WorkflowInstance workflowInstance)
+          throws WorkflowOperationException {
+
+    WorkflowOperationInstance operation = workflowInstance.getCurrentOperation();
+    String limitToOneConfig = StringUtils.trimToEmpty(operation.getConfiguration(LIMIT_TO_ONE)).toLowerCase();
```

Comment:
I would strongly urge the use of `BooleanUtils.toBoolean` here.  You still need to trim the string, but then you don't need to worry about user input being `True` vs `true`.